sydent/http/auth.py

```diff
@@ -43,8 +41,13 @@ def tokenFromRequest(request):
         token = authHeader[len("Bearer "):]
 
     # no? try access_token query param
-    if token is None and 'access_token' in request.args:
-        token = request.args['access_token'][0]
+    if token is None:
+        # The keys of request.args will be str in python2, and bytes in python3
+        token_list = request.args.get(b'access_token') or request.args.get('access_token')
```

Comment:
I believe that the `b'access_token'` will work properly in Python 2 and Python 3.

Seems like a good opportunity to add tests? 😄